#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
float64 latitude
float64 longitude
float64 altitude
float64 vel_north
float64 vel_east
float64 vel_down
float64 heading
float64 pitch
float64 roll